cts/scheduler/summary/utilization-complex.summary

```diff
@@ -31,13 +31,15 @@ Current cluster status:
 Transition Summary:
   * Stop       dummy3                              (            rhel8-1 )  due to node availability
   * Move       dummy5                              ( rhel8-2 -> rhel8-5 )
-  * Move       httpd-bundle-ip-192.168.122.131     ( rhel8-2 -> rhel8-5 )
-  * Move       httpd-bundle-podman-0               ( rhel8-2 -> rhel8-5 )
-  * Move       httpd-bundle-0                      ( rhel8-2 -> rhel8-5 )
-  * Restart    httpd:0                             (     httpd-bundle-0 )  due to required httpd-bundle-podman-0 start
-  * Start      httpd-bundle-1                      (            rhel8-1 )  due to unrunnable httpd-bundle-podman-1 start (blocked)
-  * Start      httpd:1                             (     httpd-bundle-1 )  due to unrunnable httpd-bundle-podman-1 start (blocked)
-  * Start      httpd-bundle-2                      (            rhel8-2 )  due to unrunnable httpd-bundle-podman-2 start (blocked)
+  * Stop       httpd-bundle-ip-192.168.122.131     (            rhel8-2 )  due to node availability
```

Comment:
I'm not sure what to make of these changes. I'm wary, but there seem to be about the same number of stops and starts when you account for moves. Things end up on the same node.

Reply:
Never mind, this is a bug.